app/package.json

```diff
@@ -280,7 +280,7 @@
     "unused-files-webpack-plugin": "^3.4.0",
     "url-loader": "^1.1.2",
     "uuid": "^3.4.0",
-    "web3": "1.3.4-rc.2",
+    "web3": "^1.3.3",
```

Comment:
@timoxley Using `1.3.4` gave me unnecessary warnings: https://github.com/ChainSafe/web3.js/issues/3898

Reply:
nevermind, seems to work... using `1.3.4-rc.2`